src/main/java/seedu/loyaltylift/logic/commands/AddOrderCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.loyaltylift.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.loyaltylift.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.loyaltylift.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.loyaltylift.logic.parser.CliSyntax.PREFIX_QUANTITY;
+import static seedu.loyaltylift.logic.parser.CliSyntax.PREFIX_STATUS;
+
+import seedu.loyaltylift.logic.commands.exceptions.CommandException;
+import seedu.loyaltylift.model.Model;
+import seedu.loyaltylift.model.order.Order;
+
+/**
+ * Adds a customer to the address book.
+ */
+public class AddOrderCommand extends Command {
+
+    public static final String COMMAND_WORD = "addo";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a customer to the address book. \n"
+            + "Parameters: "
+            + "1"
```

Comment:
Is this referring to a customer index? I believe we should just use 'INDEX' to be consistent with other usage messages